src/test/java/temporary/tests/TestCheckoutConfirmationPage.java

```diff
@@ -0,0 +1,49 @@
+package temporary.tests;
+
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import pages.checkoutConfirmationPage.CheckoutConfirmationPage;
+
+import static engine.drivers.WebDriverFactory.quitWebDriver;
+import static steps.CartPageSteps.goToCheckout;
+import static steps.CheckoutConfirmationPageSteps.confirmOrder;
+import static steps.MainPageSteps.*;
+import static temporary.tests.TestGuestCheckout.loginPage;
+
+public class TestCheckoutConfirmationPage extends BaseTest {
+
+    protected static CheckoutConfirmationPage checkoutConfirmationPage;
+
+    @BeforeClass
+    public static void setUp() {
+        checkoutConfirmationPage = new CheckoutConfirmationPage();
+    }
+
+    @After
+    public void tearDown() {
+        quitWebDriver();
+    }
+
+    @Test
+    public void proceedToCheckout() {
+        // Given:
+        openPage();
+        addToCart();
+        goToCart();
+        goToCheckout();
+        loginPage.selectGuestAccountOption();
+        loginPage.clickContinueButton();
```

Comment:
I assume this will be imported directly from `LoginPageSteps` once it is merged to main from my PR, right?